masonite/request.py

```diff
@@ -387,3 +387,9 @@ def send(self, params):
 
     def helper(self):
         return self
+    
+    def pop(self, *args):
```

Comment:
This seems like a lot of extra logic here. Can't the logic just be:

```python
def pop(self, *args):
    for key in args:
        if key in self.request_variables:
            del self.request_variables[key]
```

It looks to me that you are recompiling the whole list and checking if the key in not in the keys. I feel like that will be more expensive and time consuming than the solution above where we are just deleting the keys from the dictionary

Reply:
Yes, you're rigth.

Reply:
you know what ... i don't think this will work since i believe dictionaries are immutable. I wonder if there is another way to just iterate over the popped ones because sometimes there may be 20, 30+ inputs

Reply:
😕 This work, I think.

Reply:
You could try. I think you will get something along the lines of "dictionary changed during iteration" or something.

Reply:
oooh right nvm. It works because we are not iterating over the request variables, we are iterating over the args list. Got it